sdk/core/core-http/test/policies/bearerTokenAuthenticationPolicyTests.ts

```diff
@@ -61,7 +62,7 @@ describe("BearerTokenAuthenticationPolicy", function() {
     const credentialsToTest: [MockRefreshAzureCredential, number][] = [
       [refreshCred1, 2],
       [refreshCred2, 2],
-      [notRefreshCred1, 1]
+      [notRefreshCred1, 2]
```

Comment:
Well, this was actually important: https://github.com/Azure/azure-sdk-for-js/pull/10692/files#r472566282